docs/snippets/angular/button-story-component-decorator.ts.mdx

```diff
@@ -22,10 +22,6 @@ export default {
 // Note:  To see the decorator applied to your component's stories you'll need to include the template key with the component's selector.
 
 const Template: Story<ButtonComponent> = (args: ButtonComponent) => ({
-  component: ButtonComponent,
-  moduleMetadata: {
```

Comment:
Does this `moduleMetadata` also come for free now? If so, we also need to add that to MIGRATION. Let me know and I can do that.

Reply:
IMHO : no,  There is no change. It's just that it's redundant with the `component: ButtonComponent,` in `export default` 

In angular all components must be declared in only one module.
storybook/angular builds a module and declares `component` if necessary and also components listed in `moduleMetadata.declarations`.